packages/dataviews/src/dataforms-layouts/data-form-layout.tsx

```diff
@@ -47,6 +49,34 @@ export function DataFormLayout< Item >( {
 		[ form ]
 	);
 
+	const firstValidationRunRef = useRef( false );
+
+	useEffect( () => {
+		if ( firstValidationRunRef.current ) {
+			return;
+		}
+		fieldDefinitions.forEach( ( fieldDefinition ) => {
```

Comment:
Is there a reason you decided to run the validation against all the `fieldDefinitions` versus the `normalizedFormFields` ( the fields being rendered ).

This probably overlaps a 100% most of the cases, but for our `bulk` editing for example, not all the fields get rendered, but the same field definitions are still being passed in. Meaning it would still run validation for the hidden fields.

Reply:
Interesting point. I think that you're right! It doesn't make sense to run validation for all fields 👍